File: Java/Alterando.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;

public class Atualizando {

    public static void main(String[] args) {
        try {
            System.out.println("Conectando com o MySQL ...");
            Class.forName("com.mysql.jdbc.Driver");
            String url = "jdbc:mysql://localhost:3306/aula_bd";

            Connection mydb = DriverManager.getConnection(url, "root", "root");
            
            String sql = "update estado set nome = ?, sigla = ? where id = ?";
            PreparedStatement ps = mydb.prepareStatement(sql);
            ps.setString(1, "Mato Grosso");
            ps.setString(2, "MT");
            ps.setInt(3, 6);
            int registros = ps.executeUpdate();
            System.out.println("Registros afetados: " + registros);
        } catch (Exception e) {
            System.out.println("Erro: " + e);
        }
    }
}

File: Python/alterando.py
import mysql.connector

print('Conectando com o MySQL ...')

try:
    mydb = mysql.connector.connect(
        host="localhost",
        user="root",
        password="root",
        database="aula_bd"
    )
    
    sql = "update estado set nome = %s, sigla = %s where id = %s"
    estado = ("Mato Grosso", "MT", 7)

    cursor = mydb.cursor()
    cursor.execute(sql, estado)
    mydb.commit()
    print("Alterado: ", estado)
    print("Registros afetados: ", cursor.rowcount)
except BaseException as e:
    print("Erro: ", e)
else:
    mydb.close()